source/python/neuropod/tests/test_tensorflow_v2_packaging.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2021 UATC, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tensorflow as tf
+import unittest
+from testpath.tempdir import TemporaryDirectory
+
+from neuropod.packagers import create_tensorflow_neuropod
+from neuropod.tests.utils import get_addition_model_spec, check_addition_model
+from neuropod.utils.eval_utils import RUN_NATIVE_TESTS
+
+
+def create_tf_addition_model():
+    """
+    A simple addition model
+    """
+
+    class Adder(tf.Module):
+        @tf.function(
+            input_signature=[
+                tf.TensorSpec(shape=None, dtype=tf.float32),
+                tf.TensorSpec(shape=None, dtype=tf.float32),
+            ]
+        )
+        def add(self, x, y):
+            return {"out": x + y}
+
+    return Adder()
+
+
+@unittest.skipIf(tf.__version__[0] != "2", "Skipping TF 2.x tests for TF 1.x")
+class TestTensorflowV2Packaging(unittest.TestCase):
```

Comment:
Can you add tests for the 1 of logic for graph, saved_model and trackable_obj?

Reply:
`graph_def` is tested [here](https://github.com/uber/neuropod/blob/01a87fea514249a268b4ad996d477f64286a9a75/source/python/neuropod/backends/tensorflow/test/test_tensorflow_packaging.py#L67) for TF 1.x. Graph defs aren't really natural to deal with in TF 2.x (and aren't really recommended afaik) so there isn't a test for it here.